pollplace_sample.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<vip_object schemaVersion="3">
+  <source id="1">
+    <name>Department of Elections, Commonwealth of Virginia</name>
+    <vip_id>51</vip_id>
+    <datetime>2014-08-25T15:48:59</datetime>
+    <description>The Department of Elections is the official source</description>
+    <organization_url>http://www.elections.virginia.gov/</organization_url>
+    <feed_contact_id></feed_contact_id>
+    <tou_url></tou_url>
+  </source>
+  <polling_location id="80011">
+    <address>
+      <location_name>Appalachian College of Pharmacy</location_name>
+      <line1>1060 Dragon Rd</line1>
+      <line2></line2>
```

Comment:
I will change my script that generates this to treat empty strings as null.  This is something I meant to do a while ago.